src/transform/EditTransform.css

```diff
@@ -80,4 +80,11 @@ a.pagelib_edit_section_link {
 
 .pagelib-platform-android.pagelib-theme-dark.pagelib-compatibility-filter a.pagelib_edit_section_link:active:before {
   background-color: #111;
-}
\ No newline at end of file
+}
+
+.pagelib-platform-ios h3.section_heading span.pagelib_edit_section_link_container,
+.pagelib-platform-ios h4.section_heading span.pagelib_edit_section_link_container,
+.pagelib-platform-ios h5.section_heading span.pagelib_edit_section_link_container,
+.pagelib-platform-ios h6.section_heading span.pagelib_edit_section_link_container {
+    display: none !important;
```

Comment:
Agreed with your reasoning. Only really needed it with the theme changes for the inline style issue.